lib/index.js

```diff
@@ -14,9 +14,34 @@ var requireIndex = require("requireindex");
 // Plugin Definition
 //------------------------------------------------------------------------------
 
-
-// import all rules in lib/rules
-module.exports.rules = requireIndex(__dirname + "/rules");
-
-
-
+module.exports = {
+  configs: {
+    recommended: {
+      plugins: ["tree-sitter"],
+      env: {
+        "tree-sitter/globals": true
+      },
+      rules: {
+        "no-undef": "error"
+      }
+    }
+  },
+  environments: {
+    globals: {
+      globals: {
+        // globals provided by tree sitter
+        grammar: false,
+        seq: false,
+        choice: false,
+        repeat: false,
+        repeat1: false,
+        optional: false,
+        prec: false,
+        token: false,
+        alias: false
```

Comment:
@maxbrunsfeld these are the globals I found in the documentation https://tree-sitter.github.io/tree-sitter/creating-parsers#using-the-grammar-dsl
Are there any others that are part of the public API?

Reply:
I'll take that as a no, more can be added later